.github/workflows/main.yml

```
@@ -75,3 +77,20 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: EmbarkStudios/cargo-deny-action@v1
+      
+  Result:
+    name: Result
+    runs-on: ubuntu-latest
+    if: always()
+    needs:
+      - "Test"
+      - "WASM"
+      - "Lint"
+      - "Audit"
+    steps:
+      - name: Mark the job as successful
+        run: exit 0
+        if: success()
```

```suggestion
        if: needs.Test.result == 'success' && needs.WASM.result == 'success' && needs.Lint.result == 'success' && needs.Audit.result == 'success'
```

And then reverse for the below `if`?

---

Or you can invert this if, change it to `exit 1`, and remove the `exit 0` job (it's not needed as exit 0 === no step)

---

Looks like the issue was `if: always()` which can be removed in this case.